build-system/tasks/helpers.js

```diff
@@ -837,4 +845,5 @@ module.exports = {
   printNobuildHelp,
   watchDebounceDelay,
   shouldUseClosure,
+  mangleIdentifier,
```

Comment:
Why are you exporting this?

Reply:
I was going to share it with the Alan's babel plugins, but they don't seem to be committed yet.